HealthVault/Classes/Methods/MHVMethod.h

```diff
@@ -347,6 +347,13 @@ NS_ASSUME_NONNULL_BEGIN
  */
 + (MHVMethod *)updateExternalId;
 
+/**
+ Get record operations
+ 
+ @return An instance of MHVMethod.
+ */
++ (MHVMethod *)getRecordOperations;
```

Comment:
Methods are actually listed alphabetically to make them easier to find. Let's keep this pattern going.